creedengojava.properties

```diff
@@ -0,0 +1,14 @@
+category=External Analysers
+description=Provide a list of static code analyzers (for JAVA language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
+homepageUrl=https://github.com/green-code-initiative/creedengo-java
+publicVersions=2.0.0
+archivedVersions=
+
+defaults.mavenGroupId=org.green-code-initiative
+defaults.mavenArtifactId=creedengo-java-plugin
+
+2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-java"
+2.0.0.sqVersions=[9.9,10.6.0]
```

Comment:
Tell me again why you're ending compatibility early at 10.6?

Reply:
You're rigth ... I've just updated to 10.7 (after local tests with this version)...
But I can't test with 10.8, because I d'ont see a community version on docker-hub.
Do you know if a community version for 10.8 will be released soon ?
Thank you.

Last thing, what do you about my PS above ?

Reply:
The PS is fine. There is no  10.8 Community Build; it's 2024.12. Although I think 2025.1 is out, or about to be.

Reply:
OK. thank you for information.
Sonarqube has changed the version naming method to use the year and the month, isn't it ?

Reply:
Yes